docs/native/src/onboarding/onboarding.rst

```diff
@@ -22,67 +22,72 @@ Onboarding information
 The following information must be supplied to Microsoft to enable end-to-end testing and release of
 |Office iOS| integration.
 
-+----------------+-------------+--------------------------------------------+--------------------------------------------------+
-| Property       | Data Type   | Sample Value                               | Description                                      |
++-----------------+-------------+--------------------------------------------+--------------------------------------------------+
+| Property        | Data Type   | Sample Value                               | Description                                      |
```

Comment:
Looks good! I'll update the Onboarding doc & add an attachment for it here once your pull request goes through.